app/models/team_member.rb

```diff
@@ -8,4 +8,28 @@ class TeamMember < ApplicationRecord
   def name
     first_name
   end
+
+  def job_title
+    if discipline == "Development"
+      "Developer"
+    elsif discipline == "Delivery"
+      "Delivery Lead"
+    elsif discipline == "Research"
+      "Researcher"
+    elsif discipline == "Service Design"
+      "Service Designer"
+    elsif discipline == "Design"
+      "Designer"
+    elsif discipline == "Operations Engineering"
+      "Operations Engineer"
+    elsif discipline == "Strategy"
+      "Strategist"
+    elsif discipline == "Technical Architecture"
+      "Technical Architect"
+    elsif discipline == "Product"
+      "Product Manager"
+    else
+      discipline
```

Comment:
I think I had a hand in adding this else condition and I think we should have a test for the behaviour we want when the `discipline` is unexpected.